.github/workflows/lint_python.yml

```
@@ -0,0 +1,28 @@
+name: lint_python
+on:
+  pull_request:
+  push:
+  #  branches: [master]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    # strategy:
+    #  matrix:
+    #    os: [ubuntu-latest, macos-latest, windows-latest]
+    #    python-version: [3.6, 3.7, 3.8, 3.9]  # , pypy3]
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@master
+        with:
+          python-version: 3.9  # ${{ matrix.python-version }}
+      - run: sudo apt-get install libgeoip-dev -y
+      - run: pip install codespell flake8 isort pytest
+      - if: matrix.python-version >= 3.6
+        run: |
+          pip install black
+          black --check . || true
+      - run: codespell --quiet-level=2 || true  # --ignore-words-list="" --skip=""
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: isort --recursive . || true
+      - run: pip install -r requirements.txt
+      - run: pytest . || true
```

```suggestion
name: lint_python

on:
  pull_request:
  push:

jobs:
  lint_python:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@master
      - uses: actions/setup-python@master
        with:
          python-version: 3.8
      - run: sudo apt-get install libgeoip-dev -y
      - run: pip install flake8
      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
      - run: pip install -r requirements.txt
```

few comments:
1. for now python runs on py3.8 so I proposed to change it there to py3.8 too
2. current isort doesn't check anything, so it look should be `isort --check-only --diff .`, but it require additional changes, so I propose to add it in separeate PR
3. current black doesn't check anything, so as codebase py3.8+ only and also require additional changes I propose to remove matrix condition and add black in separate PR
4. current codespell doesn't check anything, but put potential issues in output, so I propose to add it in separate PR with rationalization description
5. current pytest doesn't check anything, this codebase don't have test, so I propose add tests and appropriate check in separate PR